README.md

```diff
@@ -81,6 +80,19 @@ Fb::User.new(access_token).email
  # => "john.smith@example.com"
 ```
 
+Fb::Page#insights
+---------------------
+
+For each page object created, you can get these pre-defined metrics: page_views_total, page_fan_adds_unique, page_engaged_users, page_video_views. The insights method will fetch the cumulative value of these metrics 7 days prior to two weeks ago (e.g. if today is July 6, 2017, the value of the metric will be for the 7 days prior to and ending on June 22, 2017). The selection of metrics and aggregate period will be options in a future patch.
+
+```ruby
+user = Fb::User.new 'valid_token'
+page = Fb::Page.new("name" => "test", "id" => "1234", "user" =>user).insights
```

Comment:
@S1v4 Although this code is correct, I think this would be a more common case for an example:

```ruby
user = Fb::User.new 'valid_token'
page_insights = user.pages.first.insights
```

meaning that the variable should be called `insights` and not `page` and that, commonly, you would probably pick the page from `user.pages` and no manually.